coffee_httpd/src/httpd/utils.rs

```diff
@@ -0,0 +1,29 @@
+use paperclip::actix::Apiv2Schema;
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Deserialize, Apiv2Schema, Serialize)]
+pub struct SetupRequest {
+    pub cln_dir: String,
+}
+
+#[derive(Debug, Deserialize, Apiv2Schema, Serialize)]
+pub struct InstallRequest {
+    pub plugin: String,
+    pub try_dynamic: bool,
+}
+
+#[derive(Debug, Deserialize, Apiv2Schema, Serialize)]
+pub struct RemoveRequest {
+    pub plugin: String,
+}
+
+#[derive(Debug, Deserialize, Apiv2Schema, Serialize)]
+pub struct RemoteAddRequest {
+    pub repository_name: String,
+    pub repository_url: String,
+}
+
+#[derive(Debug, Deserialize, Apiv2Schema, Serialize)]
+pub struct RemoteRmRequest {
+    pub repository_name: String,
+}
```

Comment:
```suggestion
pub mod request {
#[derive(Debug, Deserialize, Apiv2Schema, Serialize)]
pub struct SetupRequest {
    pub cln_dir: String,
}

#[derive(Debug, Deserialize, Apiv2Schema, Serialize)]
pub struct InstallRequest {
    pub plugin: String,
    pub try_dynamic: bool,
}

#[derive(Debug, Deserialize, Apiv2Schema, Serialize)]
pub struct RemoveRequest {
    pub plugin: String,
}

#[derive(Debug, Deserialize, Apiv2Schema, Serialize)]
pub struct RemoteAddRequest {
    pub repository_name: String,
    pub repository_url: String,
}

#[derive(Debug, Deserialize, Apiv2Schema, Serialize)]
pub struct RemoteRmRequest {
    pub repository_name: String,
}
}
```

and then remove the suffix Request from the name of the struct

then you have `use utils::request;` and use use the struct as `request::AddRemote`